docs/testspec/src/16_tls.rst

```diff
@@ -747,3 +747,12 @@ The tests are implemented in :srcref:`src/tests/test_tls.cpp`.
 
 .. [#boring_bogo] BoringSSL BoGo Test Suite:
                   https://github.com/google/boringssl/tree/master/ssl/test
+
+| Starting with 3.2.0, Botan runs the server test suite of TLS-Anvil
+  [#tls_anvil]_ as a nightly test. TLS-Anvil requires a HTTPS server as a peer.
+  With Botan 3.3.0 this HTTPS server is now based on Botan's Boost ASIO stream
+  wrapper. This provides a comprehensive validation of the whole TLS stack for
```

Comment:
Well, it is not required for TLS-Anvil to run against an HTTPS server; any TLS server is possible. The only reason we use the ASIO stream is the support for multiple simultaneous handshakes to speed up the test.